source/agora/consensus/EnrollmentPool.d

```diff
@@ -89,8 +89,6 @@ public class EnrollmentPool
         // check if already exists
         if (this.hasEnrollment(enroll.utxo_key, avail_height))
         {
-            log.info("Rejected already existing enrollment, Data was: {}",
```

Comment:
Why remove this?

Reply:
It is expected to happen.

Reply:
It would be good to use "Skipped" instead of "Rejected", not removing the line.